### PR TITLE
Fixes for kernel 5.11.0

### PR DESCRIPTION
**Pull request by jkinross**
Rename can_dlc2len to can_fd_dlc2len
Rename can_len2dlc to can_fd_len2dlc
Remove get_canfd_dlc

### CODE REVIEW

**Comment by sebgiles** [comment]
If it helps, I used this on `5.11.0-27-generic #29~20.04.1-Ubuntu`, no in depth testing though.

**Comment by jackthompsonnz** [comment]
Sorry just getting round to looking at this, just enabled HWE on my LTS release and came across this - will investigate now - thanks for your effort!

File: usb-to-can_socketcan/ixx_usb_fd.c [hunk 1]
```diff
@@ -614,20 +614,20 @@ static int ixx_usbfd_handle_canmsg(struct ixx_usb_device *dev,
...
 
 		if (flags & IXXAT_USBFD_MSG_FLAGS_ESI)
 			can_frame->flags |= CANFD_ESI;
 
 		can_frame->len =
-			can_dlc2len(
-			get_canfd_dlc((flags & IXXAT_USBFD_MSG_FLAGS_DLC)
-			>> 16));
+			can_fd_dlc2len((flags & IXXAT_USBFD_MSG_FLAGS_DLC)
+			>> 16);
```
Comment (jackthompsonnz) [inline usb-to-can_socketcan/ixx_usb_fd.c:627]: As per https://www.spinics.net/lists/linux-can/msg05423.html it seems socketcan simply renamed can_dlc2len to can_fd_dlc2len, unless i'm reading wrong. 

This is also related https://www.spinics.net/lists/linux-can/msg05440.html

**Comment by sebgiles** [comment]
Still going strong on my application that has been running a few hours a day. Thanks for your effort!

**Pull request merged**